src/NotesLoaderSSC.h

```diff
@@ -34,6 +34,8 @@ const float VERSION_CHART_NAME_TAG = 0.74f;
 const float VERSION_CACHE_SWITCH_TAG = 0.77f;
 /** @brief The version where note count was added as a radar category. */
 const float VERSION_RADAR_NOTECOUNT = 0.83f;
+/** @brief The version where tech stats, measure stats, and GrooveStats key were added. */
+const float VERSION_TECH_STATS = 0.84f;
```

Comment:
Looking at it again, I'm not sure if we need this. Most of these don't actually get used anywhere, they mostly just act as indicators for if/when logic needs to behave conditionally based on the ssc file version. I think I probably added this back when I was still figuring out how the cache versioning worked. 

Since we're only saving these new tags to the cache file, I don't think it's worth updating the ssc version number. Eventually I think we should make a separate writer/loader for the cache files, but that's a whole separate conversation.